ext/search_kit/Civi/Api4/Action/SearchDisplay/AbstractRunAction.php

```diff
@@ -987,7 +987,7 @@ private function getUrl(string $path, $query = NULL) {
     }
     // Use absolute urls when downloading spreadsheet
     $absolute = $this->getActionName() === 'download';
-    return \CRM_Utils_System::url($path, $query, $absolute, NULL, FALSE);
+    return (string) \Civi::url($path, 'a')->addQuery($query);
```

Comment:
@kcristiano I think you're using the `'a'` flag unconditionally here but it should depend on the value of `$absolute`